### DISCUSSION TITLE
When will the Thread Border Router add/remove prefix, is there a description about that?

### DISCUSSION THREAD

**Discussion by jinpeng1989** [openthread/openthread#8892] [Q&A] [answered]
As mentioned in other discussions, you recommend against manually configuring prefix. Thread Border Routers will automatically generate and manage their own Unique Local Prefix.  When will the Thread Border Router add/remove prefix, is there a description about that?

**Comment by jwhui** [comment, marked as answer]
See [Thread 1.3.0](https://www.threadgroup.org/support#specifications) Section 9.6 Bidirectional IPv6 Connectivity.
